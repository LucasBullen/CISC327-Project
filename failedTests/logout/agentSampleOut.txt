4a5
> Enter number of account to deposit into.
6,7c7
< Enter account number to deposit to.
< Enter amount to deposit.
---
> Enter amount (in cents) to deposit.
10c10
< Enter amount to withdraw.
---
> Enter amount (in cents) to withdraw.
