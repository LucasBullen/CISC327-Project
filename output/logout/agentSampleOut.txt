Select mode: atm or agent.
Logged into agent mode.
Enter account number to create.
Enter name of new account.
Account successfully created. Will update after synchronizing with Back Office.
Enter account number to deposit to.
Enter amount to deposit.
Deposit successful.
Enter account number to withdraw from.
Enter amount to withdraw.
Withdraw successful.
Enter account number to delete.
Enter name of account to delete.
Account successfully deleted.
Logged out.
