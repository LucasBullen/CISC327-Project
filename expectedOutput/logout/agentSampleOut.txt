Select mode: atm or agent.
Logged into agent mode.
Enter account number to create.
Enter name of new account.
Enter number of account to deposit into.
Account successfully created. Will update after synchronizing with Back Office.
Enter amount (in cents) to deposit.
Deposit successful.
Enter account number to withdraw from.
Enter amount (in cents) to withdraw.
Withdraw successful.
Enter account number to delete.
Enter name of account to delete.
Account successfully deleted.
Logged out.
